### DISCUSSION TITLE
Qualidade de download

### DISCUSSION THREAD

**Discussion by Marcelo698** [Shabinder/SpotiFlyer#545] [Q&A]
Había seleccionado 320kbs y erróneamente descarga 124kbs. Porqué?

**Comment by Marcelo698** [comment]
> Había seleccionado 320kbs y erróneamente descarga 124kbs. Porqué?

Y aunque vuelvo a configurarlo el problema persiste.

**Reply by Y2kz** [reply]
There was a recent bug fix to change the bitrate to lower quality if the audio converter fails for a quality , I think it is causing this.
Refer [#518 ](https://github.com/Shabinder/SpotiFlyer/issues/518)

**Reply by Shabinder** [reply]
Its not dependent of Audio Converter Success or Failure instead,

The app searches for the Highest quality available and if you are getting 128kbps then that means the app couldn't find a 320kbps source for the same.

**Reply by Marcelo698** [reply]
Thanks for your reply! Then I get for using Freezer
